### ISSUE TITLE
Non-obvious example

### ISSUE THREAD

**Issue by gregfenton** [prescottprue/react-redux-firebase#926]
On the queries page, section [PARSING](https://react-redux-firebase.com/docs/queries.html#parsing), the first code example:

```javascript
// Works since limitToFirst and startAt are parsed into numbers
queryParams: [`limitToFirst=${limitToFirst}`, `startAt=${startAt}`, 'orderByValue'],

// COULD CAUSE UNEXPECTED BEHAVIOR!!! Values passed to limitToFirst and startAt will remain as strings (i.e not parsed)
queryParams: ['orderByValue', `limitToFirst=${limitToFirst}`, `startAt=${startAt}`],
```
it isn't obvious how/why these two code samples differ.  Also, the fact that the prop and the variables are the same name makes it doubly less obvious what is going on here or why these two lines would behave any differently (other than the order of the array entries).

**Comment by codedpills** [comment]
I agree, it's hard to tell how or why those two code samples should work any differently considering they have the same params with similar values. Also at a first glance, it's not hard to miss the difference in positioning (assuming that matters at all).  

I think in the second instance, the orderByValue param should not be passed at all.

```javascript
// COULD CAUSE UNEXPECTED BEHAVIOR!!! Values passed to limitToFirst and startAt will remain as strings (i.e not parsed)
queryParams: [`limitToFirst=${limitToFirst}`, `startAt=${startAt}`],
```

**Comment by prescottprue** [comment]
Yeah, the example was to show that order does in fact matter, agreed that it isn't that clear

**Comment by codedpills** [comment]
> Yeah, the example was to show that order does in fact matter, agreed that it isn't that clear

@prescottprue what happens if you don't pass the orderByValue param at all? Is limitToFirst and startAt parsed or they remain as strings?
